Apps/Sandcastle/gallery/iTwin Feature Service.html

```diff
@@ -0,0 +1,126 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, maximum-scale=1, minimum-scale=1, user-scalable=no"
+    />
+    <meta
+      name="description"
+      content="Use Viewer to start building new applications or easily embed Cesium into existing applications."
+    />
+    <meta name="cesium-sandcastle-labels" content="Beginner, Showcases" />
+    <title>iTwin Feature Service</title>
+    <script type="text/javascript" src="../Sandcastle-header.js"></script>
+    <script type="module" src="../load-cesium-es6.js"></script>
+  </head>
+  <body class="sandcastle-loading" data-sandcastle-bucket="bucket-requirejs.html">
+    <style>
+      @import url(../templates/bucket.css);
+    </style>
+    <div id="cesiumContainer" class="fullSize"></div>
+    <div id="loadingOverlay"><h1>Loading...</h1></div>
+    <div id="toolbar">
+      <div id="layers"></div>
+    </div>
+    <script id="cesium_sandcastle_script">
+      window.startup = async function (Cesium) {
+        "use strict";
+        //Sandcastle_Begin
+        const serviceResponse = await fetch("https://api.cesium.com/itwin/token");
+        const { access_token: token } = await serviceResponse.json();
+
+        Cesium.ITwinPlatform.defaultAccessToken = token;
+
+        const iTwinId = "535a24a3-9b29-4e23-bb5d-9cedb524c743";
+
+        const viewer = new Cesium.Viewer("cesiumContainer");
+
+        const featureServiceBaseUrl = "https://featureservice-eus.bentley.com";
+        const proxyUrl = "http://localhost:3000/proxy";
+
+        const points = await Cesium.ITwinData.createDataSourceForRealityDataId(
+          iTwinId,
+          "60976bd9-3176-4017-974a-4fcea76346db",
+        );
+        const lines = await Cesium.ITwinData.createDataSourceForRealityDataId(
+          iTwinId,
+          "5af22b93-cf7e-4879-9305-4c6bdda7987f",
+        );
+        const areas = await Cesium.ITwinData.createDataSourceForRealityDataId(
+          iTwinId,
+          "ebec69b5-0b5f-49d8-9081-e29bcd517f6b",
+        );
+        viewer.dataSources.add(points);
+        viewer.dataSources.add(lines);
+        viewer.dataSources.add(areas);
+
+        // Create tileset of the reality data mesh
+        // TODO: swap this out with a different mesh
+        const realityMeshId = "85897090-3bcc-470b-bec7-20bb639cc1b9";
+        const realityMesh = await Cesium.ITwinData.createTilesetForRealityDataId(
```

Comment:
This tileset is really blurry for me, unless I force the `maximumScreenSpaceError` smaller.

Current `maximumScreenSpaceError: 4`:
![image](https://github.com/user-attachments/assets/41a75e2e-345c-4f3f-bd03-5e239b1786f3)


With `maximumScreenSpaceError: 1`:
![image](https://github.com/user-attachments/assets/5cc1594c-190a-4bde-861e-a75e67f285e3)

Reply:
This was a problem in the other sandcastle as well. `4` seemed to be a decent value overall but maybe not that far zoomed out. This is not the final mesh going to be used for this Sandcastle so I can re-evaluate that when I update to the other data.